packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -324,3 +347,33 @@ export interface IContextualMenuItem {
    */
   [propertyName: string]: any;
 }
+
+export interface IContextualMenuSection extends React.Props<ContextualMenu> {
+
+  /**
+   * The items to include inside the section.
+   */
+  items: IContextualMenuItem[];
```

Comment:
Yeah, I think with what you are trying to do here (not make a breaking change), you are a bit out of luck. It's probably worth at least adding a comment.

IContextualMenuItem overall is a bit of a mess, so this is a good opportunity to think about how a new 'menu' component would be structured if we decided to start working on that in the experiments package someday (of course out of scope of what you are doing now).